README.md

```diff
@@ -49,10 +49,13 @@ Run:
 
 ```bash
 docker run -it -p 8000:80 asyncapi/studio
+docker run -it -p 8000:80 -v "$(pwd)/another/path/to/asyncapi:/usr/share/nginx/html/docs/asyncapi" asyncapi/studio
 ```
 
 and then go to [http://localhost:8000](http://localhost:8000).
 
+Also feel free to check out the [docker-compose.yml](./docker-compose.yml) example.
```

Comment:
Could we paste the value of this file in the readme under the section with docker and describe how to use the studio with docker-compose? Until now, we have not had any use case that we need examples and for now I do not see the point of having them (I mean `examples` folder or something similar).

Something like:

```md
Example of using `docker-compose`:

/```yaml
services:
  async-studio:
    build: .
    container_name: events-api
    restart: on-failure
    ports:
      - "8000:80"
    volumes:
      - ./public/docs/asyncapi:/usr/share/nginx/html/docs/asyncapi # when you need to mount specification
/```
```

WDYT?